src/server/pps/server/api_server.go

```diff
@@ -1315,7 +1315,7 @@ func (a *apiServer) getLogsLoki(request *pps.GetLogsRequest, apiGetLogsServer pp
 		if err := a.env.AuthServer.CheckClusterIsAuthorized(apiGetLogsServer.Context(), auth.Permission_CLUSTER_GET_PACHD_LOGS); err != nil {
 			return errors.EnsureStack(err)
 		}
-		return lokiutil.QueryRange(apiGetLogsServer.Context(), loki, `{app="pachd"}`, time.Now().Add(-since), time.Now(), request.Follow, func(t time.Time, line string) error {
+		return lokiutil.QueryRange(apiGetLogsServer.Context(), loki, `{app="pachd"}`, time.Now().Add(-since), time.Time{}, request.Follow, func(t time.Time, line string) error {
```

Comment:
what's up with this?

Reply:
What’s really being requested here is the last `since` duration of logs. And when `request.Follow` was true, it wasn’t actually following: it requested a start time of t - since and an end time of t, then a start time of t - since + 1 ns and an end time of t, then a start time of t - since + 2 ns and an end time of t — the end time never increased. And really, for follow mode we don’t _want_ it anyway.

I briefly deleted the argument entirely, since it was never actually used for anything but `time.Now()`, but then decided it was probably better to leave it in but DTRT with a zero value, much like the Loki API itself.